pandas/tests/io/excel/test_readers.py

```diff
@@ -804,6 +804,19 @@ def test_read_excel_multiindex(self, read_ext):
         )
         tm.assert_frame_equal(actual, expected)
 
+        # GH34673: "both_name_blank_after_mi_name" sheet
```

Comment:
I would make this a separate test.  There isn't sufficient overlap (besides `mi_file`).

Reply:
Extracted into new test